<<<<<<< HEAD
# Service to Set the tracking distance of the robot while in non-blocking mode

float64 pos_dist  # mm
float64 ang_dist  # deg
---
int64 ret
string msg
=======
# Service to Set the tracking distance of the robot while in non-blocking mode

float64 pos_dist  # mm
float64 ang_dist  # deg
---
int64 ret
string msg
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
